homeassistant/components/wsdot/strings.json

```diff
@@ -0,0 +1,63 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]",
+      "invalid_travel_time_id": "WSDOT TravelTimeId does not describe a valid travel time route {travel_time_id}",
+      "no_travel_time": "Travel time route created without a travel time entity"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Your WSDOT API key."
+        },
+        "description": "Set up your Washington State Department of Transportation integration.",
+        "title": "WSDOT setup"
+      }
+    }
+  },
+  "config_subentries": {
+    "travel_time": {
+      "entry_type": "Travel time",
+      "initiate_flow": {
+        "user": "Travel time"
+      },
+      "step": {
+        "user": {
+          "data": {
+            "name": "Name"
+          },
+          "data_description": {
+            "name": "The official WSDOT description of the route."
+          },
```

Comment:
OR, we should explain a bit better, as in, "The official WSDOT description of the route" sounds like I need to enter something specific, but as it's a dropdown, we can only be specific, so maybe we should just name this `Highway` or something along those lines